### PR TITLE
PR: Add appveyor CI integration

### LINKED ISSUE #4: Add CI integration

### PR DESCRIPTION
**Pull request by goanpeca**
Fixes #4

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+install:
+  # Astropy ci-helpers. See https://github.com/astropy/ci-helpers
+  - "git clone git://github.com/astropy/ci-helpers.git"
+  - "powershell ci-helpers/appveyor/install-miniconda.ps1"
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "activate test"
+  # Install the selected Qt version
+  - "conda install --file requirements.txt -c conda-forge"
+  - "python setup.py install"
```
Comment (ccordoba12) [inline appveyor.yml]: You should use CMD_IN_ENV here. Please take a look at the Spyder yaml file.

Comment (goanpeca) [inline appveyor.yml]: yes!

**Comment by goanpeca** [comment]
@andfoy I am not getting any coverage :-| 

screen shot 2017-06-23 at 00 22 45

Also I could not run with `python setup.py develop/install` so I built the recipe and then installed that one and run pytest from that installed one.

Thoughts @ccordoba12 ?

**Comment by andfoy** [comment]
Actually, we're missing some env variables to compile using ``setuptools``, I will add them. By the way, our python packages are not including Cython sources (``pyx`` files), only C transpiled files. Should we include ``pyx`` files by default? @goanpeca  that's why coverage is not reported when you use ``conda-build``

**Comment by goanpeca** [comment]
>Actually, we're missing some env variables to compile using setuptools, I will add them. By the way, our python packages are not including Cython sources (pyx files), only C transpiled files. Should we include pyx files by default? @goanpeca that's why coverage is not reported when you use conda-build

Yes we should...

@andfoy

**Comment by goanpeca** [comment]
Also that last test seems to need to be tweaked cause now it is failing

**Comment by goanpeca** [comment]
@andfoy if you can paste as a message here the env variables needed to use setup tools, I can add them to the process.

And yes I think we need to bundle the cython sources, not only the compiled c ones

**Comment by andfoy** [comment]
@goanpeca You need to set ``LIBRARY_INC=C:\<anaconda path>\envs <If there is an env>\Library\include`` and ``LIBRARY_LIB=C:\<anaconda path>\envs <If there is an env>\Library\lib``

**Comment by andfoy** [comment]
Also, you need to compile Cython sources inplace before testing: ``python setup.py build_ext -i --compiler=msvc``

**Comment by andfoy** [comment]
Do not forget also to set ``DISTUTILS_USE_SDK=1``

**Comment by andfoy** [comment]
> And yes I think we need to bundle the cython sources, not only the compiled c ones
I will open an issue to add PYX sources to the MANIFEST.in

**Comment by goanpeca** [comment]
@andfoy actually, you need to update that on the readme... or create a BUILD.md, to give conda instructions (one line!) vs the other haha.

I prefer to use conda-build in that case, @andfoy you mean that if I include the cython sources, with MANIFEST.in it should work?

**Comment by andfoy** [comment]
> I prefer to use conda-build in that case, @andfoy you mean that if I include the cython sources, with MANIFEST.in it should work?

Let me confirm it!

**Comment by andfoy** [comment]
> @andfoy actually, you need to update that on the readme... or create a BUILD.md, to give conda instructions (one line!) vs the other haha.

I will add them, I actually forgot completely about all those variables haha

**Comment by andfoy** [comment]
By the way, do not forget to set ``CYTHON_COVERAGE=1``

**Comment by goanpeca** [comment]
Yep that was the one I did set up

File: conda.recipe/bld.bat [hunk 2]
```diff
@@ -0,0 +1,3 @@
+:: Use cython coverage on build
+"%PYTHON%" setup.py build_ext -i --compiler=msvc
```
Comment (andfoy) [inline conda.recipe/bld.bat]: This command does not install nothing at all to ``site-packages``, it only compiles all files locally

Comment (goanpeca) [inline conda.recipe/bld.bat]: Yep I just realized XD

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+install:
+  # Astropy ci-helpers. See https://github.com/astropy/ci-helpers
+  - "git clone git://github.com/astropy/ci-helpers.git"
+  - "powershell ci-helpers/appveyor/install-miniconda.ps1"
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  # Use test conda environment
+  - "activate test"
+  - "conda install --file requirements.txt -c conda-forge"
+  - "'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat'"
```
Comment (andfoy) [inline appveyor.yml]: Call? Haha

Comment (goanpeca) [inline appveyor.yml]: yes.... :-p

**Comment by goanpeca** [comment]
Ok @andfoy and @ccordoba12 this is ready, great work @andfoy :-)

I need to fix something on coveralls-python, but that is a separate issue

**Comment by andfoy** [comment]
Thanks for the great effort! @goanpeca

**Comment by ccordoba12** [comment]
Great work @goanpeca!

**Comment by andfoy** [comment]
Should I merge this? @goanpeca

**Pull request merged**